appengine/standard/firebase/firetactoe/app.yaml

```diff
@@ -0,0 +1,11 @@
+runtime: python27
+api_version: 1
+threadsafe: true
+
+handlers:
+- url: /.*
+  script: firetactoe.application
+
+libraries:
+- name: pycrypto
+  version: latest
```

Comment:
Always use explicit versions. Latest sometimes isn't the latest, and even if it were we don't want the sample to suddenly break.